src/MercadoPago/Entity.php

```diff
@@ -100,7 +100,35 @@ public static function read($params = [])
     /**
      * @return mixed
      */
-    public static function search($filters)
+    public static function all($params = [])
+    { 
+        
+        $class = get_called_class();
+        $entity = new $class();
+        $entities =  array();
+
+        self::$_manager->setEntityUrl($entity, 'list', $params);
+        self::$_manager->cleanQueryParams($entity); 
+      
+        $response = self::$_manager->execute($entity, 'get');
+      
+        if ($response['code'] == "200" || $response['code'] == "201") {
```

Comment:
why a string? maybe this is better:
```suggestion
        if (in_array($response['code'], [200, 201], true)) {
```